src/index-nomodules.css

```diff
@@ -43,18 +43,14 @@ body {
   flex-grow: 1;
   display: flex;
   flex-direction: column;
-
+  background: #f5f5f5;
```

Comment:
And that will force every page in the app to have a gray background, including the current Edit page.  That is why `.route-page-component` exists.

Reply:
Main background color of new design is gray, thus other cases are just exceptions, that can be fixes by adding additional style.
And thank you for your comment I forgot to add this style to Dialog page.

Reply:
Gray is main color of background in new design. Thus other cases is just exceptions, that can be fixed by adding additional styles.
Thank you for your comment, I forgot to add additional styles to Dialog page.

Reply:
I'm ok with that.